tests/integration/rest/diddoc/diddoc/positive_test.go

```diff
@@ -0,0 +1,202 @@
+	Expect(err).To(BeNil())
+
+	var receivedDidResolution types.DidResolution
+	Expect(json.Unmarshal(resp.Body(), &receivedDidResolution)).To(BeNil())
+	Expect(testCase.ExpectedStatusCode).To(Equal(resp.StatusCode()))
+
+	var expectedDidResolution types.DidResolution
+	Expect(utils.ConvertJsonFileToType(testCase.ExpectedJSONPath, &expectedDidResolution)).To(BeNil())
+
+	Expect(testCase.ExpectedEncodingType).To(Equal(resp.Header().Get("Content-Encoding")))
+	utils.AssertDidResolution(expectedDidResolution, receivedDidResolution)
+},
+
+	Entry(
+		"can get DIDDoc with an existent 22 bytes INDY style mainnet DID",
+		utils.PositiveTestCase{
+			DidURL: fmt.Sprintf(
+				"http://localhost:8080/1.0/identifiers/%s",
+				testconstants.IndyStyleMainnetDid,
+			),
+			ResolutionType:       testconstants.DefaultResolutionType,
+			EncodingType:         testconstants.DefaultEncodingType,
+			ExpectedEncodingType: "gzip",
+			ExpectedJSONPath:     "../../testdata/diddoc/diddoc_indy_mainnet_did.json",
+			ExpectedStatusCode:   http.StatusOK,
+		},
+	),
+
+	Entry(
+		"can get DIDDoc with an existent 22 bytes INDY style testnet DID",
+		utils.PositiveTestCase{
+			DidURL: fmt.Sprintf(
+				"http://localhost:8080/1.0/identifiers/%s",
+				testconstants.IndyStyleTestnetDid,
+			),
+			ResolutionType:       testconstants.DefaultResolutionType,
+			EncodingType:         testconstants.DefaultEncodingType,
+			ExpectedEncodingType: "gzip",
+			ExpectedJSONPath:     "../../testdata/diddoc/diddoc_indy_testnet_did.json",
+			ExpectedStatusCode:   http.StatusOK,
+		},
+	),
+
+	Entry(
+		"can get DIDDoc with an existent UUID style mainnet DID",
+		utils.PositiveTestCase{
+			DidURL: fmt.Sprintf(
+				"http://localhost:8080/1.0/identifiers/%s",
+				testconstants.UUIDStyleMainnetDid,
+			),
+			ResolutionType:       testconstants.DefaultResolutionType,
+			EncodingType:         testconstants.DefaultEncodingType,
+			ExpectedEncodingType: "gzip",
+			ExpectedJSONPath:     "../../testdata/diddoc/diddoc_uuid_mainnet_did.json",
+			ExpectedStatusCode:   http.StatusOK,
+		},
+	),
+
+	Entry(
+		"can get DIDDoc with an existent UUID style testnet DID",
+		utils.PositiveTestCase{
+			DidURL: fmt.Sprintf(
+				"http://localhost:8080/1.0/identifiers/%s",
+				testconstants.UUIDStyleTestnetDid,
+			),
+			ResolutionType:       testconstants.DefaultResolutionType,
+			EncodingType:         testconstants.DefaultEncodingType,
+			ExpectedEncodingType: "gzip",
+			ExpectedJSONPath:     "../../testdata/diddoc/diddoc_uuid_testnet_did.json",
+			ExpectedStatusCode:   http.StatusOK,
+		},
+	),
+
+	Entry(
+		"can get DIDDoc with an existent old 16 characters INDY style testnet DID",
+		utils.PositiveTestCase{
+			DidURL: fmt.Sprintf(
+				"http://localhost:8080/1.0/identifiers/%s",
+				testconstants.OldIndy16CharStyleTestnetDid,
+			),
+			ResolutionType:       testconstants.DefaultResolutionType,
+			EncodingType:         testconstants.DefaultEncodingType,
+			ExpectedEncodingType: "gzip",
```

Comment:
Yes, we are sending "Accept-Encoding" by default - `gzip, deflate, br`. When we send that Accept-Encoding (`gzip, deflate, br`) then DID Resolver returns Encoding-Type - `gzip` because we only support it for now.

Reply:
Yes, we are sending "Accept-Encoding" by default - gzip, deflate, br. When we send that Accept-Encoding (gzip, deflate, br) then DID Resolver returns Encoding-Type - gzip because we only support it for now.